### PR TITLE
Add the replica HTML/CSS for survey form

### PR DESCRIPTION
**Pull request by Kanyorok**
the form completed in the trials.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+    Hello Camper!
+
+    For now, the test suite only works in Chrome!
+    Please read the README below in the JS Editor before beginning.
+    Feel free to delete this message once you have read it.
+    Good luck and Happy Coding!
+
+    - The freeCodeCamp Team
+
+    -->
```
Comment (Kanyorok) [inline index.html:25]: I recommend removing comments from the final code. This will help to keep the code clean. 😄

File: index.html [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+          <div class="form-group">
+            <label for="name" id="name-label">Name:</label><br>
+            <input class="form-control" id="name" type="text" placeholder="Enter name here" required></input>
+            </div>
+           
+      
+            <label for="email" id="email-label">Email:</label><br>
+            <input type="email" id="email" name="email" placeholder="email@gmail.com" required >
+            <br><br>
+
```
Comment (Kanyorok) [inline index.html:46]: I suggest that you use semantic tags to lay out your HTML elements. It is a recommended practice in HTML/CSS due to compatibility with different devices, especially for people with special needs. 😃

**Review by Kanyorok — COMMENTED** [review]
✔️ Linters no errors.
✔️ Readme no Errors.

Please, make the recommended changes to the HTML file and submit it for a second review. Use the same pull request link. 😃

**Review by iambenkis — APPROVED** [review]
# Approved 👏  
### **Highlights 🚀 :heavy_check_mark: :heavy_check_mark:** 

- The repo looks professional.:heavy_check_mark:
- Git/GitHub flow best practices have been followed:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
### **Congratulations!** 🎉 
image



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @iambenkis in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**